framework/wazuh/core/batcher/client.py

```diff
@@ -0,0 +1,59 @@
+import asyncio
+from typing import Optional
+
+from wazuh.core.batcher.mux_demux import MuxDemuxQueue
+
+
+class BatcherClient:
+    """Client class to send and receive events via a MuxDemuxQueue.
+
+    Parameters
+    ----------
+    queue : MuxDemuxQueue
+        MuxDemuxQueue instance used to route messages.
+    wait_frequency: float
+        The frequency, in seconds, at which the client checks for responses in the queue.
+        Defaults to 0.1 seconds.
+
+    """
+    def __init__(self, queue: MuxDemuxQueue, wait_frequency: float = 0.1):
+        self.queue = queue
+        self.wait_frequency = wait_frequency
+
+    def send_event(self, event) -> int:
+        """Send an event through the RouterQueue.
+
+        Parameters
+        ----------
+        event : any
+            Event to send.
+
+        Returns
+        -------
+        int
+            Unique identifier assigned to the event.
+        """
+        assigned_uid = id(event)
+        self.queue.send_to_mux(assigned_uid, event)
+        return assigned_uid
+
+
```

Comment:
```suggestion

```

Reply:
Fixed in 2df43eb2a6541c8cd9b41db1381deb47fc58b3ba.